script/publish.ts

```diff
@@ -22,31 +17,49 @@ if (!releaseSHA) {
 const currentTipSHA = gitInfo.getSHA()
 if (
   !currentTipSHA ||
-  !currentTipSHA.toUpperCase().startsWith(releaseSHA.toUpperCase())
+  !currentTipSHA.toUpperCase().startsWith(releaseSHA!.toUpperCase())
```

Comment:
`releaseSHA!` can cause an error if it's `null` or `undefined`.

Reply:
I exit the script in the previous `if` statement if it’s `null` or `undefined`, but TS isn’t smart enough to pick up on that yet.